lib/model.d.ts

```diff
@@ -325,7 +325,7 @@ export interface IncludeOptions {
   /**
    * Load further nested related models
    */
-  include?: Includeable[];
+  include?: Includeable[] | IncludeAll[];
```

Comment:
Using the same test as above results in
```
test/include.ts(20,5): error TS2345: Argument of type '{ include: { all: boolean; include: { all: boolean; }[]; }[]; }' is not assignable to parameter of type 'FindOptions'.
  Types of property 'include' are incompatible.
    Type '{ all: boolean; include: { all: boolean; }[]; }[]' is not assignable to type 'Includeable[]'.
      Type '{ all: boolean; include: { all: boolean; }[]; }' is not assignable to type 'Includeable'.
        Object literal may only specify known properties, and 'all' does not exist in type 'Includeable'.
```
if I omit ` | { all: true }` from the `Includeable` type.

Not 100% sure what you mean by "if `boolean` is just `true`", but switching it to `| { all: boolean }` nets the same result as above - the incorrect version still compiles without error.